tests/python/python-neo4j-empty-password-python-test.yml

```
@@ -37,4 +37,4 @@ invalid:
     AsyncGraphDatabase,
     )
     driver = GraphDatabase.driver(uri, auth=bearer_auth(""))
-    
\ No newline at end of file
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)